sdk/src/main/java/com/silanis/esl/sdk/internal/converter/AccountRoleConverter.java

```diff
@@ -0,0 +1,37 @@
+package com.silanis.esl.sdk.internal.converter;
+
+import com.google.common.base.Optional;
+import com.silanis.esl.sdk.AccountRole;
+
+
+@SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+public class AccountRoleConverter {
+    private final Optional<AccountRole> sdkAccountRoleOptional;
+    private final Optional<com.silanis.esl.api.model.AccountRole> apiAccountRoleOptional;
+
+    public AccountRoleConverter(AccountRole sdkAccountRoleOptional) {
+        this.sdkAccountRoleOptional = Optional.fromNullable(sdkAccountRoleOptional);
+        this.apiAccountRoleOptional = Optional.absent();
+    }
+
+    public AccountRoleConverter(com.silanis.esl.api.model.AccountRole apiAccountRoleOptional) {
+        this.apiAccountRoleOptional = Optional.fromNullable(apiAccountRoleOptional);
+        this.sdkAccountRoleOptional = Optional.absent();
+    }
+
+    public AccountRole toSdkAccountRole() {
```

Comment:
Please modify `toSdkAccountRole` to `toSDKAccountRole` for the naming convention.

Reply:
Done